resources/nl/cemu.po

```diff
@@ -1,386 +1,501 @@
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.2.1\n"
+"X-Generator: Poedit 2.3.1\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: BreakpointWindow.cpp:41
+#: bootGame.cpp
+msgid "Failed to run this title because at least one of the files is damaged."
+msgstr ""
+"Kan deze game niet uitvoeren omdat in ieder geval één van de bestanden "
+"beschadigd is."
+
+#: bootGame.cpp GameUpdateWindow.cpp GeneralSettings2.cpp
+#: GraphicPacksWindow2.cpp
+msgid "Warning"
+msgstr "Waarschuwing"
+
+#: bootGame.cpp DownloadGraphicPacksWindow.cpp DumpCtrl.cpp
+#: GeneralSettings2.cpp gpu7ShaderCache.cpp helpers.cpp InputSettings.cpp
+#: MainWindow.cpp toolMemorySearcher.cpp VulkanCanvas.cpp
+#: wxCreateAccountDialog.cpp
+msgid "Error"
+msgstr "Foutmelding"
+
+#: bootGame.cpp
+msgid ""
+"Cemu detected a DLC installed at a wrong or outdated location:\n"
+"{}\n"
+"\n"
+"Do you want to move it to the correct location:\n"
+"{}"
+msgstr ""
+"Cemu heeft DLC gevonden die geïnstalleerd is in een foute of verouderde "
+"locatie:\n"
+"{}\n"
+"\n"
+"Wil je deze DLC verplaatsen naar de correcte locatie:\n"
+"{}"
+
+#: BreakpointWindow.cpp GeneralSettings2.cpp
 msgid "On"
 msgstr "Aan"
 
-#: BreakpointWindow.cpp:47 ModuleWindow.cpp:39 debugPPCThreadsWindow.cpp:43
+#: BreakpointWindow.cpp debugPPCThreadsWindow.cpp ModuleWindow.cpp
 msgid "Address"
 msgstr "Adres"
 
-#: BreakpointWindow.cpp:53 textureRelationWindow.cpp:58
+#: BreakpointWindow.cpp textureRelationWindow.cpp
 msgid "Type"
 msgstr "Type"
 
-#: BreakpointWindow.cpp:59
+#: BreakpointWindow.cpp
 msgid "Comment"
 msgstr "Opmerking"
 
-#: BreakpointWindow.cpp:201 DisasmCtrl.cpp:524
+#: BreakpointWindow.cpp DisasmCtrl.cpp
 msgid "Enter a new comment."
 msgstr "Voer een nieuwe opmerking in."
 
-#: BreakpointWindow.cpp:201
+#: BreakpointWindow.cpp
 #, c-format
 msgid "Set comment for breakpoint at address %08x"
 msgstr "Verander opmerking voor breekpunt aan adres %08x"
 
-#: BreakpointWindow.cpp:229
+#: BreakpointWindow.cpp
 msgid "Create memory breakpoint (read)"
 msgstr "Maak nieuw geheugen breekpunt (lezen)"
 
-#: BreakpointWindow.cpp:230
+#: BreakpointWindow.cpp
 msgid "Create memory breakpoint (write)"
 msgstr "Maak nieuw geheugen breekpunt (schrijven)"
 
-#: BreakpointWindow.cpp:261
+#: BreakpointWindow.cpp
 msgid "Enter a memory address"
 msgstr "Voer een nieuw geheugen adres in"
 
-#: BreakpointWindow.cpp:261
+#: BreakpointWindow.cpp
 msgid "Memory breakpoint"
 msgstr "Geheugen breekpunt"
 
-#: CemuUpdateWindow.cpp:41
+#: CemuApp.cpp
+msgid ""
+"Your mlc01 folder seems to be missing.\n"
+"\n"
+"This is where Cemu stores save files, game updates and other Wii U files.\n"
+"\n"
+"The expected path is:\n"
+"{}\n"
+"\n"
+"Do you want to create the folder at the expected path?"
+msgstr ""
+"Je mlc01 map lijkt nergens te zijn.\n"
+"\n"
+"Dit is waar Cemu de save bestanden, game updates en andere Wii U bestanden "
+"opslaat.\n"
+"\n"
+"De verwachte locatie is:\n"
+"{}"
```

Comment:
The translation for the line "Do you want to create the folder at the expected path?" is missing